### ISSUE TITLE
how to render a image with the R,t and the camera K?

### ISSUE THREAD

**Issue by ldepn** [BachiLi/redner#150]
If i have the world to cam mat (R,t) and the camera intrinsic mat K, how to render the oject?

R = torch.tensor([[0.09630630, 0.99404401, 0.05100790], 
                    [0.57332098, -0.01350810, -0.81922001], 
                    [-0.81365103, 0.10814000, -0.57120699]])
t= torch.tensor([-105.35775150, -117.52119142, 1014.87701320])

the camera intrinsic mat
fx = torch.tensor([572.41140]),
fy = torch.tensor([573.57043]),
skew = torch.tensor([0.0]),
x0 = torch.tensor([325.26110]),
y0 = torch.tensor([242.04899])

[ape.zip](https://github.com/BachiLi/redner/files/5521530/ape.zip)

I run the following code，but the img is zeros.

`
objects = pyredner.load_obj('ape.obj', return_objects=True)
intrinsic_mat= pyredner.generate_intrinsic_mat(fx = torch.tensor([572.41140]),
                        fy = torch.tensor([573.57043]),
                        skew = torch.tensor([0.0]),
                        x0 = torch.tensor([325.26110]),
                        y0 = torch.tensor([242.04899]))
         
world_2_cam = torch.tensor([[0.09630630, 0.99404401, 0.05100790, -105.35775150], 
                    [0.57332098, -0.01350810, -0.81922001, -117.52119142], 
                    [-0.81365103, 0.10814000, -0.57120699, 1014.87701320],
                    [0,0,0,1]])
cam_2_world = torch.inverse(world_2_cam).contiguous()

camera = pyredner.Camera(position = torch.tensor([0.0, 0.0, 0.0]),
                      look_at = torch.tensor([0.0, 0.0, 1.0]),
                      up = torch.tensor([0.0, 1.0, 0.0]),
                      intrinsic_mat= intrinsic_mat,
                      clip_near = 1e-2, # needs to > 0
                      resolution = (480, 640)
                      )
scene = pyredner.Scene(camera = camera, objects = objects)
scene.camera.cam_to_world = cam_2_world
img = pyredner.render_albedo(scene)
imshow(img.cpu())
'
![01](https://user-images.githubusercontent.com/15522634/98761776-25abeb80-2411-11eb-89af-61478df4d122.png)

the correct img is follwoing:

![02](https://user-images.githubusercontent.com/15522634/98763360-d2d43300-2414-11eb-8990-4ac004fbb0b5.png)

       

`

**Comment by mworchel** [comment]
Redner uses normalized device coordinates for the calibration matrix, so you have to transform it accordingly. Also, the image y-axis points downwards in the OpenCV convention but seems to point upwards in Redner (at least that's what my experiments showed).

**Comment by lith0613** [comment]
> Redner uses normalized device coordinates for the calibration matrix, so you have to transform it accordingly. Also, the image y-axis points downwards in the OpenCV convention but seems to point upwards in Redner (at least that's what my experiments showed).

@ldepn @mworchel @BachiLi  Hi,  as the example script above, can you share detailly how to transform the R,t and the camera K into the correct value in Redner ?

**Comment by hiyyg** [comment]
@ldepn @BachiLi @mworchel Have you worked out an example for this kind of rendering?
